.github/workflows/ci.yml

```diff
@@ -39,6 +39,21 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: sudo bash ./.github/scripts/install_deps.sh
+      - name: Cache cargo registry
+        uses: actions/cache@v3
+        with:
+          path: ~/ci/check/.cargo/registry
+          key: ${{ runner.os }}-ci-check-registry-${{ hashFiles('**/Cargo.lock') }}
+      - name: Cache cargo index
+        uses: actions/cache@v3
+        with:
+          path: ~/ci/check/.cargo/git
+          key: ${{ runner.os }}-ci-check-index-${{ hashFiles('**/Cargo.lock') }}
+      - name: Cache cargo build
+        uses: actions/cache@v3
+        with:
+          path: ~/ci/check/target
+          key: ${{ runner.os }}-ci-check-build-target-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
Why not use https://github.com/Swatinem/rust-cache this action? It's more simple to use.

Reply:
Fixed